content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -190,158 +214,33 @@ are available in your cluster, then Pod resource usage can be retrieved either
-```
-```
-Container Name: simmemleak
-LastState: map[terminated:map[exitCode:137 reason:OOM Killed startedAt:2015-07-07T20:58:43Z finishedAt:2015-07-07T20:58:43Z containerID:docker://0e4095bba1feccdfe7ef9fb6ebffe972b4b14285d5acdec6f0d3ae8a22fad8b2]]
-```
+Nodes have local ephemeral storage, backed either by RAM or by locally attached, writable devices. The kubelet uses local ephemeral storage to provide [`emptyDir`](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir) {{< glossary_tooltip term_id="volume" text="volumes" >}} to Pods. The kubelet also uses this kind of storage to hold [node-level container logs](/docs/concepts/cluster-administration/logging/#logging-at-the-node-level), container images, and the writable layers of running containers.
 
-You can see that the Container was terminated because of `reason:OOM Killed`, where `OOM` stands for Out Of Memory.
+"Ephemeral" means that there is no long-term guarantee about durability.
 
-## Local ephemeral storage
-{{< feature-state state="beta" >}}
+{{< warning >}}
+If a node fails, the data in its ephemeral storage can be lost. Your
+applications cannot expect any performance SLAs (disk IOPS for example)
+from local ephemeral storage.
+{{< /warning >}}
 
-Kubernetes version 1.8 introduces a new resource, _ephemeral-storage_ for managing local ephemeral storage. In each Kubernetes node, kubelet's root directory (/var/lib/kubelet by default) and log directory (/var/log) are stored on the root partition of the node. This partition is also shared and consumed by Pods via emptyDir volumes, container logs, image layers and container writable layers.
+The kubelet manages local ephemeral storage provided that the different kinds of data (for example: `emptyDir` volumes, writeable layers, container images, logs) are all placed onto the node's root filesystem. If you have a different configuration, then the kubelet does not manage local ephemeral storage and does not apply resource limits.
 
-This partition is “ephemeral” and applications cannot expect any performance SLAs (Disk IOPS for example) from this partition. Local ephemeral storage management only applies for the root partition; the optional partition for image layer and writable layer is out of scope.
+In each Kubernetes node, kubelet's root directory (`/var/lib/kubelet` by default) and log directory (`/var/log`) are part of the node's directory structure. Typically, both `/var/lib/kubelet` and `/var/log` are on the root filesystem, and the kubelet is designed with that layout in mind.
 
 {{< note >}}
-If an optional runtime partition is used, root partition will not hold any image layer or writable layers.
+If you use a separate runtime filesystem, the root partition does not hold any image layer or writable layers, and the kubelet does not manage local ephemeral storage on that node.
```

Comment:
The kubelet does manage local ephemeral storage, it just does not account for usage outside of the root partition.

Reply:
I think “root partition” means “the filesystem containing the directory where the kubelet puts the backing directories for `emptyDir` volumes” and not “the filesystem containing `/`”.

Have I got that right? If not, I need to revise.

Reply:
Yes, that is correct.